ray-on-gke/tpu/kuberay-tpu-webhook/cloudbuild.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - id: 'kuberay tpu webhook fmt'
+    name: 'golang:1.21.11'
+    dir: /workspace/ray-on-gke/tpu/kuberay-tpu-webhook
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        set -e
+        make fmt
+  - id: 'kuberay tpu webhook vet'
+    name: 'golang:1.21.11'
+    dir: /workspace/ray-on-gke/tpu/kuberay-tpu-webhook
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        set -e
+        make vet
+  - id: 'kuberay tpu webhook test'
+    name: 'golang:1.21.11'
+    dir: /workspace/ray-on-gke/tpu/kuberay-tpu-webhook
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        set -e
+        make test
+    allowFailure: false
```

Comment:
allowFailure defaults to false right?

Reply:
I wasn't sure so I included it, looks like it does though. Removed in [746f898](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/746f898bc47e3dd361dbde136b7a9fe92705b9aa).